plugins/GSdx/GSTextureCache.cpp

```diff
@@ -32,10 +32,10 @@ GSTextureCache::GSTextureCache(GSRenderer* r)
 
 	if (theApp.GetConfig("UserHacks", 0)) {
 		m_spritehack                   = theApp.GetConfig("UserHacks_SpriteHack", 0);
-		UserHacks_HalfPixelOffset      = theApp.GetConfig("UserHacks_HalfPixelOffset", 0);
-		m_preload_frame                = theApp.GetConfig("preload_frame_with_gs_data", 0);
-		m_disable_partial_invalidation = theApp.GetConfig("UserHacks_DisablePartialInvalidation", 0);
-		m_can_convert_depth            = !theApp.GetConfig("UserHacks_DisableDepthSupport", 0);
+		UserHacks_HalfPixelOffset      = !!theApp.GetConfig("UserHacks_HalfPixelOffset", 0);
+		m_preload_frame                = !!theApp.GetConfig("preload_frame_with_gs_data", 0);
+		m_disable_partial_invalidation = !!theApp.GetConfig("UserHacks_DisablePartialInvalidation", 0);
+		m_can_convert_depth            = !!(!theApp.GetConfig("UserHacks_DisableDepthSupport", 0));
```

Comment:
The `m_can_convert_depth` line doesn't actually need changed, `!!!` == `!` and already evaluates to either true or false.